.scalafix.conf

```diff
@@ -0,0 +1,10 @@
+rules = [
+    OrganizeImports
+]
+
+OrganizeImports {
+  groupedImports                    = Merge
+  coalesceToWildcardImportThreshold = 3
+  groups                            = ["re:javax?\\.", "cats.", "scala.", "org.tessellation.", "*"]
+  removeUnused                      = false
```

Comment:
I'd set that to `true`

Reply:
Unfortunately here is message when run OrganizeImports:
`ScalafixRunException: "OrganizeImports.removeUnused" is not supported on Scala 3 as the compiler is not providing enough information. Run the rule with "OrganizeImports.removeUnused" set to false to organize imports while keeping potentially unused imports.`